rivermigrate/river_migrate.go

```diff
@@ -324,9 +325,21 @@ func (m *Migrator[TTx]) migrateDown(ctx context.Context, exec riverdriver.Execut
 		return res, nil
 	}
 
-	if !opts.DryRun {
-		if _, err := exec.MigrationDeleteByVersionMany(ctx, sliceutil.Map(res.Versions, migrateVersionToInt)); err != nil {
-			return nil, fmt.Errorf("error deleting migration rows for versions %+v: %w", res.Versions, err)
+	if !opts.DryRun && len(res.Versions) > 0 {
+		versions := sliceutil.Map(res.Versions, migrateVersionToInt)
+
+		// Version 005 is hard-coded here because that's the version in which
+		// the migration `line` comes in. If migration to a point equal or above
+		// 005, we can remove migrations with a line included, but otherwise we
+		// must omit the `line` column from queries because it doesn't exist.
+		if m.line == riverdriver.MigrationLineMain && slices.Min(versions) <= 5 {
```

Comment:
The value 5 here might be clearer if put in a named constant

Reply:
Yep, good call. Changed.